addon/nodes/doc.ts

```diff
@@ -117,10 +105,12 @@ export const docWithConfig = ({
       },
     ],
     toDOM(node) {
+      console.log('rendering doc', node);
       const attrs: Record<string, unknown> = {
         lang: node.attrs['lang'] as string,
         'data-say-document': true,
         properties: node.attrs['properties'],
+        about: node.attrs['subject'],
```

Comment:
Doc nodes can define resources now? This seems like a change that is separate from the external triples thing. (just a feeling, I've maybe missed why it's here).

Reply:
fixed in [251b09a](https://github.com/lblod/ember-rdfa-editor/pull/1229/commits/251b09abe5e77eaf7605ea2627dfd2fe2ad98385)